html/about.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>The Kingdom Comes Global Worship Conference 2023</title>
+  <script
+  src="https://kit.fontawesome.com/6eed52bc92.js"
+  crossorigin="anonymous"
+></script>
+<link rel="stylesheet" href="/css/main.css" />
+</head>
+<body>
+  <!-- ==== Primary and secondary navigation menu start here ==== -->
+  <header class="main-header">
+    <div class="secondary-navbar">
+      <ul class="secondary-navbar-content flex">
+        <li>
+          <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li><a href="#"></a>Arabic</li>
+        <li><a href="#"></a>My Page</li>
+        <li><a href="#"></a>Logout</li>
+      </ul>
+    </div>
+    <nav class="primary-navbar flex">
+      <a href="index.html"
+        ><img
+          class="logo"
+          src="/images/The-Kingdom-T-Logo.png"
+          alt="The Kingdom Logo"
+      /></a>
+      <ul class="nav-menu flex">
+        <li class="nav-item">
+          <a class="nav-link active" href="#">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="index.html#program">Program</a>
+        </li>
+        <li class="nav-item"><a class="nav-link" href="#">Join</a></li>
+        <li class="nav-item">
+          <a class="nav-link" href="index.html#partners">Sponsor</a>
+        </li>
+        <li class="nav-item"><a class="nav-link" href="#">News</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Compaign</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+      </div>
+    </nav>
```

Comment:
## Kindly fix the styles with the background color also in the desktop version to make the desktop version also the same and to make the nav be shown .✅🙏🏻🚀🚀

- [ ] Great job but I think here you the nav you have the logo icon and the humburger menu but in the original design there is just a menu so please remove the logo icon from the monile version to make your website identical to the original design✅✅

- [ ] I think you forget to add the first heading above the heading so please add it to be identical to the original design. 👌🏻👌🏻

- [ ] please make the background for the about header the same color as the same as the original design so please fix it to be the same

- Your design
![image](https://user-images.githubusercontent.com/58553711/182898100-9a0780dd-c5e5-4ba3-a754-e5f79137f192.png)
 
- Original design
![image](https://user-images.githubusercontent.com/58553711/182898461-108c09e3-f882-41a6-bd95-f0957ff0d1a4.png)

Reply:
* This issue hasn't been resolved please work on it

Reply:
Sorry to ask you; could you clarify this issue; before I submitted it, I did fix it. and it is working fine on my machine and even on the live version. Please could you check it again? I did this. Thanks.

Reply:
![image](https://user-images.githubusercontent.com/52497246/183106352-ecbc60ef-9753-4dcf-a62e-36b0606bc99f.png)
## This is what i get on my end, i see some corrections were made on the live link version but here the web app still looks the same, maybe you can try to unmerge your PR let's see if that would work. `Under your repository name,` ` click Pull requests.`
`In the "Pull Requests" list, click the pull request you'd like to revert.`
`Near the bottom of the pull request, click Revert. ...`